<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStreamDialog</class>
 <widget class="QWidget" name="AddStreamDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QFormLayout" name="search_form">
       <item row="1" column="0">
        <widget class="QLabel" name="search_label">
         <property name="text">
          <string>Search Project by name or URL</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="search_text_field"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="search_button">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="accounts_dropdown"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="accounts_label">
         <property name="text">
          <string>Account</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="search_results_label">
       <property name="text">
        <string>Search Results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="search_results_list">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialog_button_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
